src/index.js

```diff
@@ -86,6 +89,14 @@ class CoursesApp extends React.Component {
               {(handleToggle) => (
                 <NavList>
                   <NavListSection>
+                    <NavListItem
+                      id="courses-app-search-item"
+                      onClick={() => {
+                        this.props.history.replace('/cr');
```

Comment:
Does this suffice to focus the search box, as requested in UICR-166?

Reply:
Hi @MikeTaylor, yes, autofocus is already implemented in past, and redirecting to this link will trigger it by default

Reply:
Excellent, thanks! (This is the kind of thing you can't so easily tell from code-reading!)